src/renderers/shared/server/ReactDOMNodeStreamRenderer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMNodeStreamRenderer
+ */
+
+'use strict';
+
+var invariant = require('fbjs/lib/invariant');
+var React = require('react');
+var ReactDOMPartialRenderer = require('ReactDOMPartialRenderer');
+
+var Readable = require('stream').Readable;
```

Comment:
I haven't found much other documentation or argumentation about `readable-stream`.

> E.g. how does it integrate with other code that assumes the native Node stream?

My understanding (which is a bit fuzzy) is that it literally is the native Node stream code pulled out of Node core. The authors are the Node Streams Working Group.

I think the benefit of using `readable-stream` would be more consistency of behavior across versions of Node, potentially making it easier to repro reported bugs. Of course, Stream hasn't changed that much since 4.x (unlike in the 0.9-0.12 timeframe), so this may not be that big a deal.

On the other hand, linking to a particular version of `readable-stream` means that someone on the React team should probably track when `readable-stream` gets released and cut new releases of React if it's useful to upgrade. If we use native `stream`, though, developers don't have to wait on the React team to get the newest version of streams; they can choose to upgrade Node themselves.

For now, I vote we use `stream`; it's a simpler setup with less maintenance involved. If we find there are a bunch of streaming bugs due to Node versions, though, we can replace it with a pinned version of `readable-stream`. Does that sound OK?